client/src/Components/General/BackButton.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+export const BackButton = ({
+  to,
```

Comment:
Could maybe give this a default value to avoid potential breakage.

Reply:
Good idea. Adding now!